rnndb/graph/gm200_texture.xml

```diff
@@ -0,0 +1,390 @@
+			<value value="0x0b" name="ETC2_RGBA"/>
+			<value value="0x19" name="EAC"/>
+			<value value="0x1a" name="EACX2"/>
+
+			<value value="0x29" name="Z24S8">
+				<doc>stencil,    no VCAA,   24-bit fixed-point depth (legacy format)</doc>
+			</value>
+			<value value="0x2a" name="X8Z24">
+				<doc>no stencil, no VCAA,   24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x2b" name="S8Z24">
+				<doc>stencil,    no VCAA,   24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x2c" name="X4V4Z24__COV4R4V">
+				<doc>no stencil, 4+4 VCAA,  24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x2d" name="X4V4Z24__COV8R8V">
+				<doc>no stencil, 8+8 VCAA,  24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x2e" name="V8Z24__COV4R12V">
+				<doc>no stencil, 4+12 VCAA, 24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x2f" name="ZF32">
+				<doc>no stencil, no VCAA,   32-bit floating-point depth</doc>
+			</value>
+			<value value="0x30" name="ZF32_X24S8">
+				<doc>stencil,    no VCAA,   32-bit floating-point depth</doc>
+			</value>
+			<value value="0x31" name="X8Z24_X20V4S8__COV4R4V">
+				<doc>stencil,    4+4 VCAA,  24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x32" name="X8Z24_X20V4S8__COV8R8V">
+				<doc>stencil,    8+8 VCAA,  24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x33" name="ZF32_X20V4X8__COV4R4V">
+				<doc>no stencil, 4+4 VCAA,  32-bit floating-point depth</doc>
+			</value>
+			<value value="0x34" name="ZF32_X20V4X8__COV8R8V">
+				<doc>no stencil, 8+8 VCAA,  32-bit floating-point depth</doc>
+			</value>
+			<value value="0x35" name="ZF32_X20V4S8__COV4R4V">
+				<doc>stencil,    4+4 VCAA,  32-bit floating-point depth</doc>
+			</value>
+			<value value="0x36" name="ZF32_X20V4S8__COV8R8V">
+				<doc>stencil,    8+8 VCAA,  32-bit floating-point depth</doc>
+			</value>
+			<value value="0x37" name="X8Z24_X16V8S8__COV4R12V">
+				<doc>stencil,    4+12 VCAA, 24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x38" name="ZF32_X16V8X8__COV4R12V">
+				<doc>no stencil, 4+12 VCAA, 32-bit floating-point depth</doc>
+			</value>
+			<value value="0x39" name="ZF32_X16V8S8__COV4R12V">
+				<doc>stencil,    4+12 VCAA, 32-bit floating-point depth</doc>
+			</value>
+			<value value="0x3a" name="Z16">
+				<doc>no stencil, no VCAA,   16-bit fixed-point depth</doc>
+			</value>
+			<value value="0x3b" name="V8Z24__COV8R24V">
+				<doc>no stencil, 8+24 VCAA, 24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x3c" name="X8Z24_X16V8S8__COV8R24V">
+				<doc>stencil,    8+24 VCAA, 24-bit fixed-point depth</doc>
+			</value>
+			<value value="0x3d" name="ZF32_X16V8X8__COV8R24V">
+				<doc>no stencil, 8+24 VCAA, 32-bit floating-point depth</doc>
+			</value>
+			<value value="0x3e" name="ZF32_X16V8S8__COV8R24V">
+				<doc>stencil,    8+24 VCAA, 32-bit floating-point depth</doc>
+			</value>
+
+			<value value="0x40" name="ASTC_2D_4X4"/>
```

Comment:
They are. Interestingly they also seem to be available on GK20A but not other Kepler chips.